style.css

```diff
@@ -0,0 +1,298 @@
+    }
+
+    .main-content
+    {
+        width: 80%;
+        margin: 0 auto;
+    }
+
+    .section-title
+    {
+        margin-top: 50px;
+        padding-left: 2%;
+    }
+
+    .test-section
+    {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        flex-wrap: wrap;
+        margin-top: 50px;
+        margin-left: 60px;
+        text-decoration: none;
+    }
+
+    .test-box
+    {
+        display: flex;
+        justify-content: flex-start;
+        align-items: center;
+        border: 1px solid #eeeeee;
+        background-color: #282828;
+        flex-wrap: wrap;
+        width: 300px;
+        height: 250px;
+        margin: 10px;
+        margin-top: 30px;
+        text-decoration: none;
+        box-shadow: 5px 5px 15px rgba(32,32,32, 0.7);
+        box-shadow: inset 0 0 10px #111111;
+        border-radius: 10px;
+    }
+
+    @media screen and (max-width: 1600px) 
+    {
+        .test-box 
+        {
+            width: 230px;
+            height: 250px;
+        }
+
+        .test-section
+        {
+            margin-left: 30px;
+        }
+    }
+
+    .test-box a
+    {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: start;
+        width: 80%;
+        margin: 0 auto;
+        
+    }
+
+    .test-box h3
+    {
+        border-bottom: 1px solid white;
+        width: 100%;
+    }
+
+    .test-box h5
+    {
+        width: 100%;
+        margin-bottom: 50px;
+        color: #cccccc;
+    }
+
+    .test-box h6
+    {
+        width: 100%;
+        margin-bottom: 10px;
+    }
+
+    .test-contents
+    {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        padding: 10px;
+        width: 300px;
+        height: 250px;
+        font-size: 20px;
+    }
+
+    .test-contents .btn-primary 
+    {
+        align-self: flex-end;
+        text-align: center;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+
+    .test-box:hover
+    {
+        transform: scale(1.05);
+        transition: transform 0.1s ease;
+        box-shadow: -3px -3px 10px rgba(128, 128, 128, 0.5);
+    }
+
+    .btn-primary {
+        background-color: darkred;
+        border-color: darkred;
+    }
+    
+    .btn-primary:hover {
+        background-color: #5e0000;
+        border-color: #5e0000;
+    }
+
+    /* Testing Page */
+
+    .test-info-section
+    {
+        background: linear-gradient(#484848, #181818);
+        margin-top: 0;
+    }
+
+    .test-info
+    {
+        height: 150px;
+        padding-left: 10px;
+        padding-top: 50px;
+        width: 80%;
+        margin: 0 auto;
+    }
+
+    .question
+     {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
+        margin-bottom: 20px;
+        max-width: 900px;
+        min-width: 900px;
+        max-height: 100px;
+        min-height: 100px;
```

Comment:
There is no need to have both `max-height` and `min-height` - just use `height`.